Creates a readme file

tapatiohaxx opened the pull request:
This readme file begins with a basic description of what this repository holds. We can eventually update it with further and more descriptive information as time goes by.

tapatiohaxx committed:
Creates a readme file

This readme file begins with a basic description of what this repository holds. We can eventually update it with further and more descriptive information as time goes by.

ZombiMigz commented on README.md:
```diff
@@ -0,0 +1,2 @@
+<h1> BroncoDirectMe Backend </h1>
+<p> This repository contains all required code to drive the BroncoDirectMe aChrome Extencion. This includes Web scrapers for retrieving all professors from RateMyProfessor.com and APIs for displaying the informatin on the extension vlient.</p>
```
looks good. We'll have more stuff later on to put. Please fix the spelling/spacing errors, thx :)

tapatiohaxx committed:
modifies readme to resolve spelling errors

This Commit adds a major heading for explaining the purpose of the app and a paragraph under dealing the function of it in greater detail. Previous heading demoted to an <code> h3 </code> subheading.

tapatiohaxx committed:
Updates the scope of the project

ZombiMigz reviewed (approved):
lgtm
